### ISSUE TITLE
Docker old version RuntimeError

### ISSUE THREAD

**Issue by Appleweed564** [visual-layer/fastdup#144]
Running this docker command on windows 11 Docker Desktop yield a RuntimeError .

`docker run -it -v /K/compare_vids/images_reference:/K/compare_vids/images_reference  karpadoni/fastdup-ubuntu-20.04:0.211 /usr/bin/python3 -c "import fastdup; fastdup.run('/K/compare_vids/images_reference/',num_images=10)"`

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/fastdup/__init__.py", line 42, in <module>
    raise RuntimeError(f"fastdup detected your are running an old version {__version__} (10 versions or more vs. the latest) please upgrade fastdup")
RuntimeError: fastdup detected your are running an old version 0.208 (10 versions or more vs. the latest) please upgrade fastdup
```

**Comment by dbickson** [comment]
Hi @Appleweed564 we have released docker version 0.915 please try it out and let us know if this works.

**Comment by Appleweed564** [comment]
Thanks @dbickson , it is working now .

I am running it on a ImageNet dataset that contains more than 1 million training images , I signed up on the visual-layer website for the free beta but did not receive a key . Should I send an email to info@visual-layer.com ?
